src/manager.cpp

```diff
@@ -347,8 +347,100 @@ types::DbusVariantType
 void Manager::collectSingleFruVpd(
     const sdbusplus::message::object_path& i_dbusObjPath)
 {
-    // Dummy code to supress unused variable warning. To be removed.
-    logging::logMessage(std::string(i_dbusObjPath));
+    try
+    {
+        // Check if BMC reaches Ready state
+        const auto l_dbusValue = dbusUtility::readDbusProperty(
```

Comment:
The commit should make the required JSON changes as well for JSONs we have under this branch.